### PR TITLE
Fixining issues

### PR DESCRIPTION
**Pull request by duskotmane**
# In this PR : 

- [ ] I added tests
- [ ] modify the README file

- Here is a link to the previous PR : [Metrics webapp PR](https://github.com/otmaneechchafyky/react-capstone-project/pull/6)

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
## Hi :raising_hand: :raising_hand: @otmaneechchafyky , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**You are not supposed to open another PR. Kindly use the same PR until your project is approved**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by FrederickMih** [comment]
Hi @otmaneechchafyky, Can you include the link to the previous PR in the summary of this PR? This way we will be able to see the comments left by the previous CR. Reviews must be consistent. Also include the reason why you decided to open another PR. Also, make sure the title of your PR is descriptive

**Review by FrederickMih — COMMENTED** [review]
Hi 🙋 🙋 @otmaneechchafyky ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
Invalid Code Review Request

You are not supposed to open another PR. Kindly use the same PR until your project is approved

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/__tests__/Info.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import Details from '../components/Info';
+
+test('renders Details component correctly', () => {
+  const { container } = render(
+    <Provider store={store}>
+      <Details />
+    </Provider>,
+  );
+  expect(container.firstChild).toMatchSnapshot();
+});
```
Comment (OyePriscilla) [inline src/__tests__/Info.test.js:14]: - [ ] Please kindly read through this [React Documentation](https://redux.js.org/usage/writing-tests) as regards writing tests. Integration test and unit tests for pure functions in your Redux code are still missing. ℹ️ 

Untitled

File: src/reportWebVitals.js
Comment (OyePriscilla) [inline src/reportWebVitals.js:1]: - [ ] Please kindly remove all the broiler template that was generated by CRA. Below screenshot is a guide I think. 👍 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @otmaneechchafyky 👍, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Engaging presentation ✔️ 
Well formatted `README` file ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @otmaneechchafyky  👋 

### Things to appreciate
- [ ] Implemented the features requested by the previous code reviewers. 👍 

### [OPTIONAL]
- [ ] As it's a React Capstone project, you were supposed to use Gitflow and not GitHub flow.

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**